content/blog/2023/07/10/2023-07-10-jenkins-june-newsletter.adoc

```diff
@@ -0,0 +1,140 @@
+* IPv6 support for get.jenkins.io and every other public services, thanks to their migration to a brand new AKS cluster with dual stack enabled.
+
+
+[[modern-ui]]
+image:/images/post-images/2023/01/12/jenkins-newsletter/ui_ux.png[User Experience Update]
+
+Contributed by: author:markewaite[Mark Waite]
+
+User experience improvements continue thanks to the efforts of link:https://github.com/mawinter69[Markus Winter], author:janfaracik[Jan Faracik], link:https://github.com/jenkinsci/jenkins/pulls/meiswjn[Jan Meiswinkel], and others.
+
+Markus is improving the look and feel of the delete dialog, so that it will be consistent and will appear within the web page as a modal dialog, rather than appearing outside the page as a browser dialog.
+
+Jan Faracik continues to improve the look and feel of Jenkins pages, including recent improvements to the logs page and improvement in the cascading style sheets.
+
+
+[[platform]]
+image:/images/post-images/2023/01/12/jenkins-newsletter/platform-modernization.png[Platform Modernization Update]
+
+Contributed by: author:gounthar[Bruno Verachten]
+
+Several platform updates occurred throughout June:
+
+* CentOS 7: Early End of Life announcement.
+Read the link:/blog/2023/05/30/operating-system-end-of-life[blog post] for details.
+Included in version link:/changelog/#v2.407[2.407].
+
+* Docker Image Updates
+** Jenkins agent and controller images: Utilizing link:https://www.updatecli.io/[updatecli] for image management.
+JDK17 version tracking for `ssh-agent` and Alpine Linux and JDK version tracking for the controller.
+** `ssh-agent`: Released versions link:https://github.com/jenkinsci/docker-ssh-agent/releases/tag/5.6.0[`5.6.0`] and link:https://github.com/jenkinsci/docker-ssh-agent/releases/tag/5.4.0[`5.4.0`] with JDK version tracking and other enhancements.
+** `docker-agent`: New release link:https://github.com/jenkinsci/docker-agent/releases/tag/3131.vf2b_b_798b_ce99-2[`3131.vf2b_b_798b_ce99-2`] with updated dependencies and `3.18.2` Alpine Linux version.
+** `docker-inbound-agent`: Release link:https://github.com/jenkinsci/docker-inbound-agent/releases/tag/3131.vf2b_b_798b_ce99-2[`3131.vf2b_b_798b_ce99-2`] with updated parent image.
+** Docker Controller: Release link:https://github.com/jenkinsci/docker/releases/tag/2.411[`2.411`] with Jenkins 2.411 and security policy enhancements. Check out the link:https://github.com/jenkinsci/docker/pull/1647[release notes] for more details.
+* Docker Hub Stats
+** Monthly image exports shared on link:https://docs.google.com/spreadsheets/d/1NfGpKDXaRQh1DRD64CG1fY6CoIG9D--H8Ft01VhfzRQ/edit#gid=256200265[this spreadsheet].
+** ArchLinux usage is low, but not deprecated.
+* Work in Progress
+** Windows image availability for the controller. Latest version: `2.410`.
+
+
+[[documentation]]
+image:/images/post-images/2023/02/07/2023-02-07-jenkins-newsletter/documentation.png[Documentation Update]
+Contributed by: author:markewaite[Mark Waite]
```

Comment:
```suggestion
Contributed by: author:kmartens27[Kevin Martens]
```
I'm back submitting the documentation update, so just a fix for that!

Reply:
Ooopsie, my bad. 😊 
Sorry about that.